code/src/styling/lib/Buttons.js

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+
+export const DirectionButton = styled.button`
```

Comment:
Soo cool styling! Love that the color fades out 😀

Reply:
It kind of makes me think they turn disabled 🙈 Would it make more sense if they faded in (turned green)? 😇